apps/bridge-dapp/src/containers/ShieldedAssetsTableContainer/ShieldedAssetsTableContainer.tsx

```diff
@@ -108,32 +122,72 @@ const columns: ColumnDef<ShieldedAssetDataType, any>[] = [
       </Typography>
     ),
   }),
-
-  columnHelper.accessor('assetsUrl', {
-    header: 'Action',
-    cell: () => {
-      return (
-        <div className="flex items-center space-x-1">
-          <Button variant="utility" size="sm" className="p-2">
-            <AddBoxLineIcon className="!fill-current" />
-          </Button>
-
-          <Button variant="utility" size="sm" className="p-2">
-            <SendPlanLineIcon className="!fill-current" />
-          </Button>
-
-          <Button variant="utility" size="sm" className="p-2">
-            <WalletLineIcon className="!fill-current" />
-          </Button>
-        </div>
-      );
-    },
-  }),
 ];
 
 export const ShieldedAssetsTableContainer: FC<
   ShieldedAssetsTableContainerProps
 > = ({ data = [], onUploadSpendNote }) => {
+  const onTransfer = useCallback(() => {
+    console.warn('Transfer is not implemented yet');
+  }, []);
+
+  const onDeposit = useCallback(() => {
+    console.warn('Deposit is not implemented yet');
+  }, []);
+
+  const onWithdraw = useCallback(() => {
+    console.warn('Withdraw is not implemented yet');
+  }, []);
```

Comment:
We will need to open issues for each operation. Essentially, when a user selects a quick options we decode the note and pre-populate the operation with the applicable data.